subsys/net/l2/wifi/Kconfig

```diff
@@ -27,3 +27,8 @@ config WIFI_MGMT_RAW_SCAN_RESULTS_ONLY
 	  to the application.
 
 endif # WIFI_MGMT_RAW_SCAN_RESULTS
+
+config WIFI_MGMT_FORCED_PASSIVE_SCAN
```

Comment:
This can now be removed or moved to driver Kconfig. Or if you think its useful for all drivers, then we can use it to hardcode passive scan mode in wifi_mgmt.

Reply:
Yeah, moved to driver.